File: PRAK404/PRAK404-2210817310015-RIDHANISETIADI.py
while True:   
    print(" program :")
    print("1. Penjumlahan")
    print("2. Pengurangan")
    print("3. Perkalian")
    print("4. Pembagian")
    print("5. Exit")
    program = int(input("Pilih program yang mau dimasukkan :"))
    if program == 1:
        nilai1 = int(input("Masukkan nilai 1 :"))
        nilai2 = int(input("Masukkan nilai 2 :"))
        penjumlahan = nilai1 + nilai2
        print("Hasil nilai1 %d + nilai2 %d adalah : %d"%(nilai1,nilai2,penjumlahan))
    elif program == 2:
        nilai1 = int(input("Masukkan nilai 1 :"))
        nilai2 = int(input("Masukkan nilai 2 :"))
        pengurangan = nilai1 - nilai2
        print("Hasil nilai1 %d - nilai2 %d adalah : %d"%(nilai1,nilai2,pengurangan))
    elif program == 3:
        nilai1 = int(input("Masukkan nilai 1 :"))
        nilai2 = int(input("Masukkan nilai 2 :"))
        perkalian = nilai1 * nilai2
        print("Hasil nilai1 %.2f * nilai2 %.2f adalah : %.2f"%(nilai1,nilai2,perkalian))
    elif program == 4:
        nilai1 = int(input("Masukkan nilai 1 :"))
        nilai2 = int(input("Masukkan nilai 2 :"))
        pembagian = nilai1 / nilai2
        print("Hasil nilai1 %d / nilai2 %d adalah : %d"%(nilai1,nilai2,pembagian))
    elif program == 5:
        print("terima kasih telah menggunakan kalkulator ridhani setiadi")
        break
    else :
        print("Input anda salah, silahkan coba lagi")
    
        

File: PRAK404/PRAK404-2210817310015-RIDHANISETIADI.c
#include<stdio.h>
int main(void){
    for(int i=1;i<=1;i++){
    int i,pilihan;
    float a,b,hasil;
menu :
    printf("\n1.Penjumlahan\n");
    printf("2.Pengurangan\n");
    printf("3.Perkalian\n");
    printf("4.Pembagian\n");
    printf("5.Exit\n");
     printf("Masukkan pilihan :");
    scanf("%d",&pilihan);
    if (pilihan==5){
    printf("Terimakasih, telah menggunakan kalkulator Ridhani Setiadi",hasil);
    break;
    }
    else if (pilihan>5){
    printf("Input anda salah, silahkan coba lagi");
    goto menu ;
    }
    printf("Masukkan Nilai Pertama :");
    scanf("%f",&a);
    printf("Masukkan Nilai Kedua :");
    scanf("%f",&b);
if(pilihan==1){
    hasil=a + b;
    printf(" Hasil penjumlahan antara %.f+%.f = %.2f",a,b,hasil);
    goto menu ;
}
else if(pilihan==2){
    hasil=a - b;
    printf(" Hasil pengurangan antara %.f - %.f = %.2f",a,b,hasil);
    goto menu ;
}
else if(pilihan==3){
    hasil=a * b;
    printf(" Hasil perkalian antara %.2f * %.2f = %.2f",a,b,hasil);
    goto menu ;
}
else if(pilihan==4){
    hasil=a / b;
    printf(" Hasil pembagian antara %.f / %.f = %.2f",a,b,hasil);
    goto menu ;
}  
}
}


